src/DIRAC/WorkloadManagementSystem/Utilities/RemoteRunner.py

```diff
@@ -0,0 +1,129 @@
+""" RemoteRunner
+
+Mostly called by workflow modules, RemoteRunner is generally the last component to get through before
+the script/application execution on a remote machine.
+Depending on an environment variable WORKLOADEXECLOCATION, it decides whether it should take care of the execution.
+RemoteRunner has been designed to send script/application on remote worker nodes having no outbound connectivity
+(e.g. supercomputers)
+"""
+import os
+import shlex
+import time
+
+from DIRAC import gLogger, gConfig, S_OK, S_ERROR
+from DIRAC.Core.Utilities.Subprocess import systemCall
+from DIRAC.Core.Security.ProxyInfo import getProxyInfo
+from DIRAC.Core.Utilities.LockRing import LockRing
+from DIRAC.Resources.Computing.ComputingElementFactory import ComputingElementFactory
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getQueue
+from DIRAC.WorkloadManagementSystem.Client import PilotStatus
+
+
+class RemoteRunner(object):
+    def __init__(self):
+        self.log = gLogger.getSubLogger("RemoteRunner")
+        self.workloadExecutionLocation = os.environ.get("WORKLOADEXECLOCATION")
```

Comment:
Can't it read `gConfig.getValue("/LocalSite/WorkloadExecLocation")`? What is the reason for a new env variable?

Reply:
Now I don't remember, seems possible, I will try to make the change.